lib/truffle/truffle/patching.rb

```diff
@@ -78,4 +78,21 @@ def activate
     end
   end
 
+  # Allows specifying patching outside the context of RubyGems by setting an environment variable value.
+  # E.g., setting TRUFFLERUBY_CUSTOM_PATCH='launchy:lib,spec' will allow patching of files in the 'lib' and 'spec'
+  # of a local checkout of the 'launchy' gem.
```

Comment:
`spec` seems like a bad example, unless you want to patch the specs?
The default with gem activation is `full_require_paths`, which in most cases I think only `lib`.

Reply:
It would be much better to already absolute paths like:
```
TRUFFLERUBY_CUSTOM_PATCH="launchy:`pwd`/lib,`pwd`/spec"
```
Otherwise it will break as soon as truffleruby is invoked with a different CWD.

Reply:
This is functionality being added to strictly address an internal testing concern. I'll deal with the fallout if it breaks. If I make it a fully qualified path, it will fail the `$LOAD_PATH` search and I really don't want to complicate that process.

Reply:
To address your other point, I would indeed like to have the ability to patch specs. This can be helpful for those that use `fork`, for instance.